Library/Homebrew/formula_installer.rb

```diff
@@ -780,12 +777,6 @@ def sanitized_argv_options
       args << "--devel"
     end
 
-    formula.options.each do |opt|
-      name = opt.name[/^([^=]+)=$/, 1]
-      value = self.args&.value(name) if name
-      args << "--#{name}=#{value}" if value
-    end
-
```

Comment:
Not sure about this one, I guess this was previously used inside formulae using `ARGV.value`, which doesn't work anymore. Am I missing something here?

Reply:
I'm OK with deleting it if you are and we can re-add if it causes issues.

Reply:
Yeah, I think we can come up with a better way to refactor this *if* this still used and the way it is used.